### PR TITLE
New Trait: Parkour Training

### PR DESCRIPTION
**Pull request by angelofallars**
# Description

**Parkour Training** is a 3-point Physical trait that increases your table climbing speed (and all other climbables), and crawling speed, and reduces the cooldown on laying down/standing up.

Inspired by the SS13 trait called Freerunning, expanding on it by including new mechanics around the Laying Down system.

Stats (Adjustable):
- 30% faster table climbing speed
- 25% faster crawling speed
- Cooldown on laying down/standing up reduced from 2.5 seconds to 2 seconds

## Media

image

# Changelog

:cl: Skubman
- add: Add Parkour Training, a 3-point trait that makes you faster with climbing tables and crawling.

### CODE REVIEW

File: Resources/Locale/en-US/traits/traits.ftl [hunk 1]
```diff
@@ -55,3 +55,10 @@ trait-name-Foreigner = Foreigner
 trait-description-Foreigner =
     For one reason or another you do not speak this station's primary language.
     Instead, you have a translator issued to you that only you can use.
+
+trait-name-ParkourTraining = Parkour Training
+trait-description-ParkourTraining =
+    Whether as a hobby, lifestyle, or professional training, you are trained in the discipline of parkour.
+    You climb 25% faster over tables, railings, fences and other similar structures.
+    You crawl 25% faster.
+    You can lie down and stand up in succession 10% faster.
```
Comment (DEATHB4DEFEAT) [inline Resources/Locale/en-US/traits/traits.ftl:64]:
```suggestion
    You climb, crawl, and lie faster
```

Comment (DangerRevolution) [reply Resources/Locale/en-US/traits/traits.ftl:64]: image

Comment (angelofallars) [reply Resources/Locale/en-US/traits/traits.ftl:64]: I forgor to make it not a draft

**Review by DangerRevolution — CHANGES_REQUESTED** [review]
make an opposite trait IMO

File: Resources/Locale/en-US/traits/traits.ftl (hunk 1 above)
Comment (angelofallars) [reply Resources/Locale/en-US/traits/traits.ftl]: Resolved with [75e35bc](https://github.com/Simple-Station/Einstein-Engines/pull/670/commits/75e35bc6190daa7d803efb25f20c26bb886f2741)

> Whether as a hobby, lifestyle, or professional training, you are trained in the discipline of parkour.
> You're faster with climbing, crawling, lying down, and getting up.

File: Content.Server/Traits/Assorted/LayingDownModifierComponent.cs
Comment (DangerRevolution) [inline Content.Server/Traits/Assorted/LayingDownModifierComponent.cs:1]: Need to check, not sure this works.

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

**Comment by angelofallars** [comment]
> make an opposite trait IMO

Maybe in a separate PR, once I figure out the proper way to set movement speed with a trait.

**Review by DangerRevolution — APPROVED** [review]

**Review by VMSolidus — APPROVED** [review]

**Pull request merged**